pkg/kubectl/plugins/examples/kubectl-foo

```diff
@@ -6,16 +6,10 @@ if ARGV[0] == "metadata"
   "use"    : "foo",
   "short"  : "Foo plugin short description",
   "long"   : "Foo really LONG description\\n\\nCan take multi lines.",
-  "tunnel" : true
+  "tunnel" : false
 }
 META
   puts metadata
 else
-  api_host = ENV['KUBECTL_PLUGIN_API_HOST']
-  api_endpoint = "http://#{api_host}/api"
-
   puts "I'm the magnificent foo plugin."
-  puts ""
-  puts "I can use the API from #{api_host}:"
-  puts `curl -s #{api_endpoint}`
```

Comment:
I liked this example, b/c it was simple and quite self-explanatory, why removing it?